lib/help.js

```diff
@@ -388,6 +388,42 @@ class Help {
     return argument.description;
   }
 
+  /**
+   * @param {string} title
+   * @param {string[]} items
```

Comment:
Do we need a `helper` too?

Reply:
Yes, thanks. (I am slightly disappointed `npm run check` did not pick this up!)